wyoming_whisper/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM python:3.9-bullseye as build
```

Comment:
We should be using our base images. Can we use Alpine maybe?

Reply:
This uses [ctranslate2](https://pypi.org/project/ctranslate2/) under the hood, which may build on Alpine but I'm using the pre-compiled wheels for now.